qpc/source/edit.py

```diff
@@ -56,6 +56,11 @@ def __init__(self, subparsers):
         self.parser.add_argument('--port', dest='port',
                                  metavar='PORT', type=validate_port,
                                  help=_(messages.SOURCE_PORT_HELP))
+        self.parser.add_argument('--satellite_version',
```

Comment:
I think English language learners and myself would rejoice if we could refer to Satellite as "sat" but I understand that may not be clear enough.